packages/api-server/utils/statusDict.js

```diff
@@ -0,0 +1,6 @@
+const STATUS = {
+  UNVERIFIED: 'UNVERIFIED',
+  VERIFIED: 'VERIFIED',
+  NORMAL: 'NORMAL',
```

Comment:
what does it mean?

Reply:
UNVERIFIED and VERIFIED is the status for customers, NORMAL is the status for accounts. Those status constants will be more clear in the next commit